### PR TITLE
f-restaurant-card@v0.13.0 - delivery fee and minimum order

### PR DESCRIPTION
**Pull request by jamieomaguire**
v0.13.0
------------------------------
*January 17, 2022*

### Added
- RestaurantFees component to display delivery fee and minimum order
- RestaurantFees unit tests
- Default background colour to restaurant card image
### Changed
- Changed restaurant image role from "img" to "presentation"
- Added RestaurantFees data to storybook
- Updated f-vue-icons dependency for new cash small icon

Screenshot 2022-01-17 at 14 03 39

### CODE REVIEW

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    display: flex;
+    align-items: center;
+}
+
+.c-restaurantCard-fees-item {
+    position: relative;
+    padding-right: spacing();
+    margin-right: spacing(x0.5);
+    white-space: nowrap;
+    font-size: $font-body-s-paragraph * 1px;
```
Comment (ashleynolan) [inline packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue]: You can use the font-size mixin to convert this to relative units:

`@include font-size($font-body-s-paragraph);`

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    &:before {
+        display: none;
+    }
+
+    &:after {
+        content: '\2022';
```
Comment (ashleynolan) [inline packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue]: Always worth commenting any special chars, just so it's a bit clearer

```suggestion
        content: '\2022'; // round bullet character
```

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue (hunk 1 above)
Comment (jamieomaguire) [reply packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue]: cheers!

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue (hunk 2 above)
Comment (jamieomaguire) [reply packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue]: good point - ty!

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+.c-restaurantCard-fees-item {
+    position: relative;
+    padding-right: spacing();
+    margin-right: spacing(x0.5);
+    white-space: nowrap;
+    @include font-size($font-body-s-paragraph);
+
+    &:before {
```
Comment (sonnyprince) [inline packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue]: Is this required?

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        right: 0;
+        height: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+    }
+
+    &:last-of-type {
+        padding-right: 0;
```
Comment (sonnyprince) [inline packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue]: If you removed the absolute positioning from the `after` (maybe used lineheight and or translates to position on relative) and added the margin/padding to that you would'nt need to set and then remove padding/margin on the parent elm :)

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue (hunk 3 above)
Comment (jamieomaguire) [reply packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue]: which part, sorry?

File: packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue (hunk 4 above)
Comment (jamieomaguire) [reply packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantFees/RestaurantFees.vue]: This is actually far more graceful, thanks for the suggestion! Just commiting the update now :)

**Review by sonnyprince — APPROVED** [review]

**Review by dawidchar — APPROVED** [review]

**Pull request merged**